wiki/fr/fr-Central-Servers.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: wiki
+title: "Serveurs centraux"
+lang: "fr"
+permalink: "/wiki/Central-Servers"
+---
+
+# Enregistrement du serveur public
+
+En mai 2020, afin de minimiser le problème de la saturation du serveur central, nous avons créé plusieurs serveurs centraux basés sur le genre musical (chacun pouvant lister 150 serveurs au maximum). 
+
+Lorsque vous démarrez un serveur public, vous verrez un message vous indiquant si votre serveur a été enregistré avec succès ou non. Si vous laissez votre serveur en fonction, il réessaiera de s'inscrire jusqu'à ce qu'un créneau soit libéré.  
+
```

Comment:
Same than above about "enregistré". The original English word is "listed". We can keep "listé" here too which is what I recommand.

All good for the rest.